apps/coordinator/src/components/CreateAddress/HardwareWalletPublicKeyImporter.jsx

```diff
@@ -160,7 +160,7 @@ const HardwareWalletPublicKeyImporter = ({
           </Button>
         </Box>
         <InteractionMessages
-          messages={interaction.messagesFor({ state: status })}
+          messages={interaction().messagesFor({ state: status })}
```

Comment:
javascript was a mistake lmao

Reply:
You're right. Typescript is the solution! 🎉 

Getting there in caravan.